### PR TITLE
downloads: add PELUX 3.0

### PR DESCRIPTION
**Pull request by sashko**
Signed-off-by: Oleksandr Kravchuk <oleksandr.kravchuk@pelagicore.com>

### CODE REVIEW

File: download.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ title: Downloads
...
 
-## PELUX 2.0
-As of 2018-07-02, PELUX 2.0 has been released! The docs can be found
-[here](//pelux.io/software-factory/PELUX-2.0/).
+## PELUX 3.0
+As of 2018-11-15, PELUX 3.0 has been released! The docs can be found
+[here](//pelux.io/software-factory/PELUX-3.0/).
 
 ### Main features
  - Derived from poky
```
Comment (thenor) [inline download.md]: I'm guessing there will be more changes done to this page before the WIP flag is removed.

**Comment by thenor** [comment]
I would also add in the commit message that the PELUX 2.0 downloads page is moved to a new page, so that the latest release can be on the start page.

File: download.md [hunk 2]
```diff
@@ -42,30 +40,29 @@ As of 2018-07-02, PELUX 2.0 has been released! The docs can be found
...
-* [pelux.xml](https://github.com/Pelagicore/pelux-manifests/blob/v2.0/rocko/pelux.xml)
+* [pelux.xml](https://github.com/Pelagicore/pelux-manifests/blob/v3.0/sumo/pelux.xml)
   in PELUX Yocto Manifests repository, see below.
 * The version of [PELUX Software Development
-  Environment](https://github.com/Pelagicore/pelux-sde/tree/v2.0) as pointed out
-  below.
-* The version of [PELUX Software Factory Baseline](//pelux.io/software-factory/PELUX-2.0/) as
+  Environment](https://github.com/Pelagicore/pelux-sde/tree/v3.0) as pointed out below.
+* The version of [PELUX Software Factory Baseline](//pelux.io/software-factory/PELUX-3.0/) as
   pointed out below.
```
Comment (fhajredini) [inline download.md]: Probably its a good idea to add links here such as:

- [Qemu Artifacts](https://pelux.io/artifacts/archive/artifacts_qemu-x86-64_nogfx/) 
- [Intel minimal images](https://pelux.io/artifacts/archive/artifacts_intel/) 

And so on.. But maybe its best to wait until these are actually ready.

Comment (thenor) [reply download.md]: @fhajredini Have your changes already been done? Shouldn't the addition of the links you mention be done after you are finished?

File: download.md [hunk 3]
```diff
@@ -4,9 +4,9 @@ title: Downloads
 permalink: /downloads/
 ---
 
-## PELUX 2.0
-As of 2018-07-02, PELUX 2.0 has been released! The docs can be found
-[here](//pelux.io/software-factory/PELUX-2.0/).
+## PELUX 3.0
+As of 2018-11-16, PELUX 3.0 has been released! Documentation can be found
```
Comment (fhajredini) [inline download.md]:
```suggestion
As of 2018-11-19, PELUX 3.0 has been released! Documentation can be found
```

**Pull request merged**